### ISSUE TITLE
Continuation of "How to Use This Tool" issue

### ISSUE THREAD

**Issue by starikcetin** [game-ci/unity-activate#94]
I was asked to create another issue, so here it is, continuing from https://github.com/game-ci/unity-activate/issues/77#issuecomment-1210511875

I have filled them all:

![image](https://user-images.githubusercontent.com/21353964/183885470-d819a5a3-8527-4b58-b582-ab65b0ff72dc.png)

This is for personal license if that helps with any context.

**Comment by webbertakken** [comment]
The code you're hitting literally checks if the value is passed or not 

https://github.com/game-ci/unity-activate/blob/33cd3ea96f11f3a1ddcf8fa9d8d265d5ea81e240/dist/entrypoint.sh#L13

The env variable is (directly) passed from the action into the docker image here:

https://github.com/game-ci/unity-activate/blob/33cd3ea96f11f3a1ddcf8fa9d8d265d5ea81e240/src/model/docker.ts#L26

I don't see any reason why that would suddenly stop working.

Could you try echoing your UNITY_LICENSE variable from secrets?

**Comment by starikcetin** [comment]
OK I figured it out. It looks like we are supposed to pass the contents of the `.ulf` file and not the `.alf` file to `UNITY_LICENSE` secret. It is working now. Thank you.

**Comment by webbertakken** [comment]
To keep it factual: it would have given a different error if you had provided the `.alf` file.

And you're correct, the contents of the `.ulf` file are what are needed for the activation using this action.

**Comment by starikcetin** [comment]
I *did* pass the contents of the `.alf` file to `UNITY_LICENSE` secret before starting to write the workflow for the manual activation. That was the output I got when I first executed it. The second and third executions (without changing the secrets, mind you) gave a different output:

```
Requesting activation (personal license)
###########################
#         Failure         #
###########################

Please note that the exit code is not very descriptive.
Most likely it will not help you solve the issue.

To find the reason for failure: please search for errors in the log above.

Exit code was: 1
Error: The process '/usr/bin/docker' failed with exit code 1
```

**Comment by webbertakken** [comment]
> Requesting activation (personal license)

Sounds right.

I'm unable to explain why the secrets didn't pass the second and third execution. The code in this action doesn't have any conditional code beyond checking if the variable being set or not. If you didn't change the secret then I would go as far as to say that GitHub may not have propagated the change to all machines somehow.

**Comment by starikcetin** [comment]
Possible, maybe there is an undocumented delay in when the secrets are made available to the jobs after creation, or maybe I have hit an edge case.
